### PR TITLE
Bump Rails & Ruby, remove Travis

### PR DESCRIPTION
**Pull request by carlosantoniodasilva**
Just some minor quality of life upgrades:

- Rails: 6.1.3.2 (helps clear up the current security warning about Rails)
- Ruby: 2.7.3
- Remove Travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +0,0 @@
-language: ruby
```
Comment (carlosantoniodasilva) [inline .travis.yml:1]: Is there a reason we still have travis setup with GitHub Actions?

Comment (m5o) [reply .travis.yml:1]: Only legacy. Since GitHub Actions is stable now, it’s safe to sunset Travis.

**Review by m5o — APPROVED** [review]
Thank you

💅 These are my habits, on rails updates I always check RailsDiff to review framework changes. 
Eg. https://railsdiff.org/6.1.2.1/6.1.3.2

**Comment by carlosantoniodasilva** [comment]
Thanks @m5o.

I usually take a look at the changelogs and that kind of stuff, but forgot about railsdiff, that's always good to check especially on minor/major updates :+1:.

**Pull request merged**